### DISCUSSION TITLE
Encountered difficulties with NVM

### DISCUSSION THREAD

**Discussion by SnowDreamXUE** [coreybutler/nvm-windows#1174] [General] [answered]
After downloading Vue using a node managed by nvm, Vue is not recognized.I've set the NODE_HOME and two system variables for NVM in the system environment variables, and I still can't solve it
![Uploading {74B468D9-E18A-4A69-9E3C-10590915097C}.png…]()
![Uploading {2F5995E8-E293-460F-A4CF-86B9C25B2DA4}.png…]()

**Comment by SnowDreamXUE** [comment]
![{2F5995E8-E293-460F-A4CF-86B9C25B2DA4} - 副本](https://github.com/user-attachments/assets/688dcf3c-0d90-49b8-af62-5b2946a7229f)
![{74B468D9-E18A-4A69-9E3C-10590915097C} - 副本](https://github.com/user-attachments/assets/bef1ea0e-7e3c-4818-8ca6-0f4f1f1f093c)

**Reply by coreybutler** [reply]
I suspect you need to remove pretty much everything except `NVM_HOME` and `NVM_SYMLINK`. Use `nvm debug` to identify conflicting paths. See https://github.com/coreybutler/nvm-windows/wiki/Common-Issues#why-do-i-need-to-uninstall-nodejs-before-installing-nvm-for-windows.

I'm not a Vue user, so I don't know if it is designed for global use or only for project use. What I do know is if you want any module to be available globally, you have to run `npm install -g <module>`. That's what creates the cmd file in the npm bin directory that is required to make a module run.

**Reply by SnowDreamXUE** [reply, marked as answer]
My Vue module was installed globally using npm normally, but it didn't work. I did the same after restarting my computer. But when I  double-click the vue.cmd file in the node_global directory to run it, everything was running normally again.
![image](https://github.com/user-attachments/assets/82099b8c-99ef-4dd9-8636-d3d6caad8e20)
Finally, thank you for your help
